base/shared/src/main/scala/scalaz/typeclass/Group.scala

```diff
@@ -0,0 +1,16 @@
+package scalaz
+package typeclass
+
+/* A Group is a Monoid with negation and inverse, such that
+ * a |+| inverse(a)  = monoid.empty
+ * inverse(a) |+| a  = monoid.empty
```

Comment:
Aren't these `Group` laws? @vmarquez

Reply:
these are the laws, as per @jdegoes  comment https://github.com/scalaz/scalaz/issues/1526#issuecomment-347657494 we will define laws later on

Reply:
Yeah. I wrote this after not realizing that "Does Group have any laws?" was intended to mention "Magma" instead.